### PR TITLE
Added  requested version for csstype

### PR DESCRIPTION
**Pull request by saraswati-patra**
## Checklist

- [x] Have you checked and followed all the points mention in the [CONTRIBUTING.MD](https://github.com/ppc64le/build-scripts/blob/master/CONTRIBUTING.md)
- [x] Did you run the script(s) on fresh container with `set -e` option enabled and observe success ?
- [ ] Did you have **Legal approvals** for patch files ?

### CODE REVIEW

**Comment by seth-priya** [comment]
@saraswati-patra  is the inclusion of the "csstype_RHEL7.6.sh.bak" file in this commit intentional?

**Comment by saraswati-patra** [comment]
> @saraswati-patra is the inclusion of the "csstype_RHEL7.6.sh.bak" file in this commit intentional?

Added .bak file wrongly,Deleted and commit.thanks

File: c/csstype/csstype_RHEL7.6.sh [hunk 1]
```diff
@@ -2,11 +2,13 @@
...
 #
 # Package       : csstype
-# Version       : v2.6.0
-# Source        : https://github.com/frenic/csstype
-# Tested on     : RHEL 7.6
-# Node Version  : v12.19.1
-# Maintainer    : Vedang Wartikar <vedang.wartikar@ibm.com>
+# Version       : v2.6.0,v2.6.9
+# Source repo   : https://github.com/frenic/csstype
+# Tested on     : RHEL 7.6,8.5
```
Comment (seth-priya) [inline c/csstype/csstype_RHEL7.6.sh:7]: this should be RHEL 7.6, RHEL 8.5 for consistency

**Review by seth-priya — CHANGES_REQUESTED** [review]

**Comment by seth-priya** [comment]
@vathsalashetty25 @kumarvikas2020 if @saraswati-patra  can't , can one of you handle this?

**Review by seth-priya — APPROVED** [review]

**Pull request merged**